_data/team_members.yml

```diff
@@ -8,6 +8,17 @@
     - Master, Sciences de la matière, <a href="https://www.ens-lyon.fr">ENS Lyon, France</a>
     - PhD, Physics, <a href="https://quantuminstitute.yale.edu/">Yale Quantum Institute</a>
     - Postdoc, <a href="https://www.rle.mit.edu/qp">MIT/Harvard</a>
+- name: Guus Avis
+  photo: Guus1.jpg
+  info: Postdoctoral Researcher
+  time: started August 2023
```

Comment:
It seems this field does not actually make it to the final web page, but I still added it for consistency.
Is this on purpose or an oversight @Krastanov ?

Reply:
an oversight, but we do not need to worry about it in this PR